doc/source/new-project-guide/level-3-tutorial.rst

```diff
@@ -198,6 +198,22 @@ Copy and paste the value while replacing ``<path-to-your-workspace-folder>`` wit
      # For Windows (Powershell) user
     $env:PYTHONPATH = "$env:PYTHONPATH;<path-to-your-workspace-folder>"
 
+To avoid retyping this command every time you open a new terminal, you can add it to your shell configuration file:
+
+.. code-block:: bash
```

Comment:
Don't we also let them know where the config files are located? 

for macOS, it's located in  `~/.zshrc` and then typing `source ~/.zshrc` is required

Reply:
`~/.basrc` etc., do z-shell, bash and powershell to cover most cases.